spec/yaml/swiftnav/sbp/signing.yaml

```diff
@@ -51,6 +51,23 @@ definitions:
             desc: nanoseconds of second
                   (range 0-999999999)
 
+  - ECDSASignature:
+      short_desc: ECDSA signature
+      fields:
+        - len:
```

Comment:
I think if we move `len` field to after `data` then we technically would not need to rev `MSG_ECDSA_SIGNATURE` since the wire format/order/length of the bytes would need to be the same, but not sure that the message checker in CI would be able to handle that.